specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -5330,6 +5811,70 @@
           "description": "Error when What-If operation fails."
         }
       }
+    },
+    "Tags": {
+      "type": "object",
+      "description": "key and value pairs for tags",
+      "properties": {
+        "tags": {
+          "type": "object"
+        }
+      }
+    },
+    "TagPatchRequest": {
+      "type": "object",
+      "description": "Tag Request for Patch operation.",
+      "properties": {
+        "operation": {
+          "type": "string",
+          "description": "The operation type for the patch api.",
+          "enum": [
+            "Replace",
+            "Merge",
+            "Delete"
+          ]
+        },
+        "properties": {
+          "$ref": "#/definitions/Tags",
+          "description": "tags object passing in the request."
+        }
+      }
+    },
+    "TagRequest": {
+      "type": "object",
+      "description": "Tag Request for Put, Get, Delete operations.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/Tags",
+          "description": "tags object passing in the request."
+        }
+      }
+    },
+    "TagResponse": {
+      "type": "object",
+      "description": "Tag Response for Patch, Put, Get, Delete operations.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The ID of the tag response."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the tag response."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "object",
+          "description": "The type of the tag response."
+        },
+        "properties": {
```

Comment:
can you try to make `properties` the only required thing?

Reply:
Added required field.